### ISSUE TITLE
The parameters and Flops that I ran are different from those given in the official documentation.

### ISSUE THREAD

**Issue by jsJSjs666** [ultralytics/ultralytics#17547]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

`                from  n    params  module                                       arguments
  0                  -1  1       464  ultralytics.nn.modules.conv.Conv             [3, 16, 3, 2]
  1                  -1  1      4672  ultralytics.nn.modules.conv.Conv             [16, 32, 3, 2]
  2                  -1  1      7360  ultralytics.nn.modules.block.C2f             [32, 32, 1, True]
  3                  -1  1     18560  ultralytics.nn.modules.conv.Conv             [32, 64, 3, 2]
  4                  -1  2     49664  ultralytics.nn.modules.block.C2f             [64, 64, 2, True]
  5                  -1  1     73984  ultralytics.nn.modules.conv.Conv             [64, 128, 3, 2]
  6                  -1  2    197632  ultralytics.nn.modules.block.C2f             [128, 128, 2, True]            
  7                  -1  1    295424  ultralytics.nn.modules.conv.Conv             [128, 256, 3, 2]
  8                  -1  1    460288  ultralytics.nn.modules.block.C2f             [256, 256, 1, True]
  9                  -1  1    164608  ultralytics.nn.modules.block.SPPF            [256, 256, 5]                  
 10                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']
 11             [-1, 6]  1         0  ultralytics.nn.modules.conv.Concat           [1]
 12                  -1  1    148224  ultralytics.nn.modules.block.C2f             [384, 128, 1]
 13                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']
 14             [-1, 4]  1         0  ultralytics.nn.modules.conv.Concat           [1]
 15                  -1  1     37248  ultralytics.nn.modules.block.C2f             [192, 64, 1]                   
 16                  -1  1     36992  ultralytics.nn.modules.conv.Conv             [64, 64, 3, 2]
 17            [-1, 12]  1         0  ultralytics.nn.modules.conv.Concat           [1]
 18                  -1  1    123648  ultralytics.nn.modules.block.C2f             [192, 128, 1]
 19                  -1  1    147712  ultralytics.nn.modules.conv.Conv             [128, 128, 3, 2]
 20             [-1, 9]  1         0  ultralytics.nn.modules.conv.Concat           [1]
 21                  -1  1    493056  ultralytics.nn.modules.block.C2f             [384, 256, 1]                  
 22        [15, 18, 21]  1    751507  ultralytics.nn.modules.head.Detect           [1, [64, 128, 256]]            
myYOLOv8 summary: 225 layers, 3011043 parameters, 3011027 gradients, 8.2 GFLOPs 
`
When I ran the code, the parameters printed were 3011043 and the FLOPs was 8.2G.However, the parameters given in the official website document are 3.2M and FLOPs is 8.7G.Does anyone know why?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @jsJSjs666, thank you for bringing this query to our attention and contributing to Ultralytics 🚀! It's important to ensure consistency between your findings and our documentation.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) that can help us investigate and understand this discrepancy better.

If your question is related to custom training or configurations ❓, please provide additional context, such as the specific model configurations you used, to assist us in troubleshooting the issue.

We also recommend checking out our [Docs](https://docs.ultralytics.com) for comprehensive guides and examples that might clarify the differences you're observing.

Stay connected with the community for more insights:
- Join our conversation on [Discord](https://ultralytics.com/discord) 🎧.
- Participate in threads on our [Discourse](https://community.ultralytics.com).
- Explore discussions on our [Subreddit](https://reddit.com/r/ultralytics).

## Upgrade

Make sure you’re running the latest version of `ultralytics` to get all the recent updates and bug fixes. Update using the command below:

```bash
pip install -U ultralytics
```

## Environments

You can try running your setup in one of these verified environments to confirm the results:

- [Google Colab](https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb) <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab">
- [Paperspace Gradient](https://console.paperspace.com/github/ultralytics/ultralytics) <img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient">
- [Kaggle Notebooks](https://www.kaggle.com/models/ultralytics/yolo11) <img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle">
- [Docker Image](https://hub.docker.com/r/ultralytics/ultralytics) <img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls">

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

This is an automated response, but an Ultralytics engineer will review your issue soon to assist you further. If the CI badge above is green, it means all our tests are passing, ensuring the software's robustness across various modes and tasks.

**Comment by Y-T-G** [comment]
The original has 80 classes. Yours seem to have only 1 class.

**Comment by jsJSjs666** [comment]
> The original has 80 classes. Yours seem to have only 1 class.

Yes，I used it to solve a binary classification problem.Are parameters and FLOPs related to classes?

**Comment by glenn-jocher** [comment]
Yes, the number of classes directly affects both parameters and FLOPs in YOLO models. The detection head's parameters scale with the number of classes since it needs to predict probabilities for each class. With 80 classes (COCO) vs 1 class, you'll see different parameter counts and computational requirements, which explains the discrepancy between your values and the documentation's reported metrics.
